microcosm_postgres/encryption/v2/column.py

```diff
@@ -121,3 +181,7 @@ def unencrypted(self, **kwargs: Any) -> Mapped[T | None]:
             ),
             **kwargs,
         )
+
+    def beacon(self, **kwargs: Any):
+        # TODO - assume all encrypted fields need to use beacons
```

Comment:
do we still need this TODO? not sure that's a valid assumption.

Reply:
nope - will strip. 👍